src/main/java/com/ss/ita/google/ui/pages/runnerAndProperties/PropertiesProvider.java

```diff
@@ -11,9 +11,9 @@
 
     public PropertiesProvider() throws IOException {
 
-            this.fileInputStream = new FileInputStream("src/test/resources/data.properties");
-            this.properties = new Properties();
-            properties.load(fileInputStream);
+        this.fileInputStream = new FileInputStream("src/test/resources/data.properties");
```

Comment:
where's the property file?

Reply:
its just code format changes, property file is in main branch in src/test/resources/data.properties